hack/pre-release.sh

```diff
@@ -185,11 +188,29 @@ switch_git_branch $base_branch $new_branch
 # Locate and store the WMCO version from the makefile 
 # finds "WMCO_VERSION ?=" and selects text following it
 WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
+
 update_WMCO_version $stream $WMCO_VERSION
 update_OCP_version $stream $WMCO_VERSION 
+
+# check to see if the opeator-sdk version is correct 
+# if not, update it to the correct version 
+operator_sdk_current_version=$(operator-sdk version |  grep -oP 'operator-sdk version: "\K[^"]+' )
+if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
+  echo "incorrect version. operator-sdk version must be $OPERATOR_SDK_VERSION"  
+  echo "downloading and installing operator-sdk version $OPERATOR_SDK_VERSION"
+
+  export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+  export OS=$(uname | awk '{print tolower($0)}')
+
+  export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/$OPERATOR_SDK_VERSION
+  curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+
+  chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
```

Comment:
I suggest just exiting if the SDK version does not match. We don't want to be overwriting the SDK version without user permission.

Reply:
Moreover you are looking at the OS and architecture. So how can you assume `/usr/local/bin` across OSes?